src/main/java/world/bentobox/bentobox/database/objects/Island.java

```diff
@@ -562,7 +562,7 @@ public int getMaxEverProtectionRange() {
     public void setMaxEverProtectionRange(int maxEverProtectionRange) {
         this.maxEverProtectionRange = maxEverProtectionRange;
         if (maxEverProtectionRange > this.range) {
-            maxEverProtectionRange = this.range;
+            this.maxEverProtectionRange = this.range;
```

Comment:
This is the main bug fix.

Reply:
:D nice 

This is one of the reasons why I always use `this.` for class variables :D